.github/workflows/PR.yml

```diff
@@ -0,0 +1,33 @@
+name: Master Pull Request
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - uses: c-hive/gha-npm-cache@v1
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Lint
+        run: npm run lint
+      
+      - name: Linting failed (attempting fix...)
```

Comment:
I think this step can be removed for now as it makes no sense without the git-auto-commit action.

Reply:
True, will update and remove later today, good catch!

Reply:
Just added a commit action for this actually since if the fixer fails CI won't commit anyway but if it passes, we don't pass it to the user to fix locally and then re-push. Should make it easy for contributors to get it right more often than not since most issues would be minor styling topics, etc I would think.